weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1430,11 +1437,11 @@ def _select_objs_query(
                         ) AS rn
                     FROM object_versions
                     WHERE project_id = {{project_id: String}}
+                       AND {conditions_part}
```

Comment:
this should go next to `WHERE rn = 1` i think. else your versions will be off if you include digest in this condition set